docs/guides/0x-animate-your-visualizations.md

```diff
@@ -0,0 +1,391 @@
+
+```js
+viz.variables.animation.play();
+```
+
+#### Add control elements
+
+The next step is to add buttons and elements that allow you to control and interact with the animation. 
+
+The   elements we will add are:
+
+* **Progress** slider to display the animation progress.
+* **Play** button to play or resume the animation.
+* **Pause** button to pause the animation.
+* **Duration** slider to control the speed of the animation.
+
+```html
+<aside>
+    <header>
+        <h1>Animation controls</h1>
+    </header>
+    <section>
+        <p>Progress: <input type="range" id="js-progress-range" min="0" max="1" step="0.01"></p>
+    </section>
+    <section>
+        <button id="js-play-button">Play</button>
+        <button id="js-pause-button">Pause</button>
+        <input type="range" id="js-duration-range" min="0" max="30" step="1">
+    </section>
+</aside>
+```
+
+As seen above, there is an `id` attribute assigned to  each element giving us access to each one using JavaScript:
+
+```js
+const $progressRange = document.getElementById('js-progress-range');
+const $playButton = document.getElementById('js-play-btn');
+const $pauseButton = document.getElementById('js-pause-btn');
+const $durationRange = document.getElementById('js-duration-range');
+```
+
+> Note: this guide makes use of some conventions. When setting an `id` to an element that is going to be accessed via JavaScript, the `id` starts with `js`. In addition, when assigning the element (`const $progressBanner = document.getElementById('js-progress-banner')`), the JavaScript value starts with `$`, which indicates that it contains an HTML element.
+
+#### Add listening events 
+
+In this step, we will add listening events tied to the different  interaction buttons used  to control the animation. For example, we will tell the **Pause** button that it has to be ready and react when clicked  to pause the animation.
+
+```js
+$pauseButton.addEventListener('click', () => {
+    viz.variables.animation.pause();
+});
+```
+
+And the same applies to the **Play** button:
+
+```js
+$playButton.addEventListener('click', () => {
+    viz.variables.animation.play();
+});
+```
+
+In the same way, we can  update the **duration** of the animation when the duration range is adjusted in the slider:
+
+```js
+$durationRange.addEventListener('change', () => {
+    viz.variables.duration = parseInt($durationRange.value, 10);
+});
+```
+
+#### Update the progress
+
+You have learned how change the animation by interacting with the different buttons, and now you are going learn how to update the current progress of the animation. This function will be the one responsible of updating the progress range by updating the range value with the result of `getProgressPct()` method:
+
+```js
+function updateProgress () {
+    $progressRange.value = viz.variables.animation.getProgressPct();
+}
+```
+
+Let’s call this function periodically by using `setInterval`:
+
+```js
+setInterval(updateProgress, 100);
```

Comment:
The controls are great, but I miss here the **Date** itself!, maybe displayed in a simple label. 
I miss it so badly... 😄

Reply:
Added a currentTime label